usage/src/main/java/com/cloud/usage/UsageManagerImpl.java

```diff
@@ -186,6 +186,7 @@ public class UsageManagerImpl extends ManagerBase implements UsageManager, Runna
     private Future _heartbeat = null;
     private Future _sanity = null;
     private boolean  usageSnapshotSelection = false;
+    private static TimeZone usageTimeZone = TimeZone.getTimeZone("GMT");
```

Comment:
@winterhazel, this refer to the aggregation time zone, right? If so, it should be renamed to something more intuitive, like `usageAggregationTimeZone`.

Reply:
Yup, I have renamed it.